sorc/ncep_post.fd/CALHEL3.f

```diff
@@ -407,7 +407,6 @@ SUBROUTINE CALHEL3(LLOW,LUPP,UST,VST,HELI)
 !       COMPUTE STORM-RELATIVE HELICITY
 !
 !!$omp  parallel do private(i,j,n,l,du1,du2,dv1,dv2,dz,dz1,dz2,dzabv,ie,iw,jn,js,z1,z2,z3)
-      DO N=1,2 ! for dfferent helicity depth
```

Comment:
@EdwardColon-NOAA  Are you aware how line 410 was introduced previously? What does "different helicity depth mean?